lib/elixir/lib/kernel.ex

```diff
@@ -932,6 +937,11 @@ defmodule Kernel do
   Removes the first occurrence of an item on the left list
   for each item on the right.
 
+  The complexity of `a -- b` is proportional to `length(a) * length(b)`,
+  meaning that it will be very slow if both `a` and `b` are long lists.
```

Comment:
I think we should not have absolute terms but more relative: "meaning it will be slower as both lists grow in size."